### PR TITLE
Added price field to the Products resource.

### PR DESCRIPTION
**Pull request by WesJourdan**
I would have left-justified the input text for price, but it looks like the NumberInput form is hard coded to be right justified.

### CODE REVIEW

File: mobile/Yote/package.json [hunk 1]
```diff
@@ -11,8 +11,8 @@
     "lodash": "^4.13.1",
     "moment": "^2.13.0",
     "prop-types": "^15.6.0",
-    "react": "^16.0.0",
-    "react-native": "^0.50.3",
+    "react": "^16.3.2",
```
Comment (grantfowler) [inline mobile/Yote/package.json:14]: did you change the package dependencies on purpose? if so, why?

File: client/modules/product/components/ProductForm.js.jsx [hunk 2]
```diff
@@ -48,6 +48,14 @@ function ProductForm({
               rows={3}
               placeholder="This is where the content goes..."
             />
+            <NumberInput
```
Comment (grantfowler) [inline client/modules/product/components/ProductForm.js.jsx:51]: when creating a new product, you will get an error as the page loads, "Failed prop type: The prop `value` is marked as required in `NumberInput`, but its value is `undefined`." could you try to figure out why this is and where to set a default value for the field (0)? it should be very similar to where 'title' and 'description' get their defaults (the empty string '').

File: mobile/Yote/package.json (hunk 1 above)
Comment (WesJourdan) [reply mobile/Yote/package.json:14]: That was unintentional. I was making a mess of things trying to get the mobile part running, I missed that when I was cleaning it up.

File: client/modules/product/components/ProductForm.js.jsx (hunk 2 above)
Comment (WesJourdan) [reply client/modules/product/components/ProductForm.js.jsx:51]: Definitely! Fixing it now.

**Comment by ffugly** [comment]
Part one looks good.

**Pull request closed without merging**